examples/persistent-volumes/volumes/local-01.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolume
+apiVersion: v1beta3
+metadata:
+  name: pv0001
+  labels:
+    type: local
+spec:
+  capacity:
+    storage: 10Gi
```

Comment:
Agreed.  I originally had those 3 (gleaned from the cloud storage APIs) but was encouraged to use just size for the first version.  I introduced ResourceRequirements.Requests in this PR, which can grow to accommodate more than just storage.  Similarly, the PersistentVolumeManager can attempt to match on more than just size.